diff --git a/src/components/room/widgets/avatar-info/menu/AvatarInfoWidgetAvatarView.tsx b/src/components/room/widgets/avatar-info/menu/AvatarInfoWidgetAvatarView.tsx
--- a/src/components/room/widgets/avatar-info/menu/AvatarInfoWidgetAvatarView.tsx
+++ b/src/components/room/widgets/avatar-info/menu/AvatarInfoWidgetAvatarView.tsx
@@ -315,9 +315,6 @@
                     <ContextMenuListItemView onClick={ event => processAction('ambassador_alert') }>
                         { LocalizeText('infostand.button.alert') }
                     </ContextMenuListItemView>
-                    <ContextMenuListItemView onClick={ event => processAction('ambassador_kick') }>
-                        { LocalizeText('infostand.button.kick') }
-                    </ContextMenuListItemView>
                     <ContextMenuListItemView onClick={ event => processAction('ambassador_mute') }>
                         { LocalizeText('infostand.button.mute') }
                         <FaChevronRight className="right fa-icon" />
